README.md

```diff
@@ -1,4 +1,147 @@
 [![CI](https://github.com/aoscloud/aos_core_iam_cpp/workflows/Build/badge.svg)](https://github.com/aoscloud/aos_core_iam_cpp/actions?query=workflow%3ABuild)
 [![codecov](https://codecov.io/github/aoscloud/aos_core_iam_cpp/branch/main/graph/badge.svg?token=kg8h7ATd9S)](https://codecov.io/github/aoscloud/aos_core_iam_cpp)
 
-# aos_core_iam_cpp
+# Aos core IAM manager
+
+## Prepare build environment
+
+```sh
+sudo apt install lcov libsofthsm2 libsystemd-dev
+pip install conan
+```
+
+# Build for host
+
+A simple script is created to build a manager for the host system
+
+```sh
+./host_build.sh
+```
+
+It installs all external dependencies to conan, creates `./build` directory and builds the manager with unit tests and coverage calculation target.
+
+Cmake options:
+
+| Option | Description |
+| --- | --- |
+| `WITH_TEST` | creates unit tests target |
+| `WITH_COVERAGE` | creates coverage calculation target |
+| `WITH_DOC` | creates documentation target |
+
+Options should be set to `ON` or `OFF` value.
+
+Cmake variables:
+
+| Variable | Description |
+| --- | --- |
+| `CMAKE_BUILD_TYPE` | `Release`, `Debug`, `RelWithDebInfo`, `MinSizeRel` |
+| `CMAKE_INSTALL_PREFIX` | overrides default install path |
+
+## Run unit tests
+
+Configure example:
+
+```sh
+cd ${BUILD_DIR}
+conan install ../conan/ --output-folder . --settings=build_type=Debug --build=missing
```

Comment:
Is it required if we use `host_build.sh`?

Reply:
no, it is required when cmake options are different from the ones in ./host_build.sh

Reply:
Then it should be stated that it is required if you need to change options. And worth to mention which options are set by ./host_build.sh